### PR TITLE
Working copy of v3

### PR DESCRIPTION
**Pull request by apandada1**
This works. However,
1. ~~Need to implement the system tray (this can probably be done without `sni-qt`, using native features of PySide2).~~ (done)
2. Need to implement setup.py, so that it can be submitted to Debian.
3. ~~ć (in Vasović) does not show up correctly in the actual program, although it shows up correctly inside Qt Creator.~~
4. This supports global menu in KDE (previously it did not), but it does not seem to support Kvantum themes (may be that's not implemented in Pyside itself)
5. I could not figure out how to get past the byte/str error in `check_displays.py` and wrote a few lines of code which is not very elegant (commit 4b7fca7). We might need to rewrite this file. 

I don't see any point in porting v3 to older Ubuntu versions since there are no new features, and v2 works perfectly fine. It will be made available in 20.04 onwards (it's best if we can implement systray and setup.py, and get it into Debian before April).
@LordAmit Please try to fix 5.
Also, `pyside2-uic` still makes files for Python2. One has to port the code to Python 3 (either using `2to3` tool or the handy https://pythonconverter.com/)

### CODE REVIEW

**Comment by apandada1** [comment]
**We are ready for the release**!! _All features_ of v2 have been implemented.
@LordAmit To test, install PySide2 from pip3. I will figure out the exact dependencies for Debian packaging in the meanwhile.
However, Pyside 2 does not seem to follow GTK themes and Kvantum themes correctly.
The left side shows how most other programs look. Also, v3 has exactly the same look in Linux Mint Cinnamon (with those blue colors in the slider), while v2 correctly mimicked the GTK theme. 
compare

**Comment by LordAmit** [comment]
Good work; I need you to change the version from 3 to 2.2 though. This is not a major release because all we are doing is making it compatible with present frameworks. This requires work, of course. But major releases come with new features or has something major going on 😄

**Comment by apandada1** [comment]
Ok, I will do that. I named it version 3 because we are now using python 3.

On Fri, Jan 17, 2020, 8:09 PM Amit Seal Ami <notifications@github.com<mailto:notifications@github.com>> wrote:

Good work; I need you to change the version from 3 to 2.2 though. This is not a major release because all we are doing is making it compatible with present frameworks. This requires work, of course. But major releases come with new features or has something major going on 😄

**Comment by apandada1** [comment]
Reverted back to 2.3

**Comment by LordAmit** [comment]
good catch, I forgot that we already were 2.2 😛

**Comment by apandada1** [comment]
Same theming issue in 20.04 XFCE.
xfce

**Comment by apandada1** [comment]
@LordAmit Which new features are you planning to implement in v3?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Brightness Controller
 
-This is version 2 of Brightness Controller. Please note that it is compatible with python2 only. It supports an arbitrary number of displays!
+This is version 3 of Brightness Controller, ported to Python 3 and PySide 2. It supports an arbitrary number of displays!
```
Comment (apandada1) [inline README.md]: @LordAmit Does this support an arbitrary no. of displays?

Comment (LordAmit) [reply README.md]: If I remember correctly, yes. It lists all your devices under primary and secondary so that you can control them as you want.

**Comment by apandada1** [comment]
The issue of not detecting themes is not there in 19.10, where PySIde2 is available from the repo. It was due to installing PySide2 from pip.

**Pull request merged**